src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -75,6 +75,11 @@ object JesBackend {
       eis.exists(ei => ei.key == JesBackend.InfoKeys.JesRunId && ei.value.isDefined)
   }
 
+  private val IsTransient: (Execution, Seq[ExecutionInfo]) => Boolean = (e: Execution, eis: Seq[ExecutionInfo]) => {
```

Comment:
Could you add a mini-comment above this to say what `transient` means in this context? It seems to be closer to `AlreadyRunning`